kalite/facility/templates/facility/facility_user.html

```diff
@@ -71,7 +71,17 @@ <h3 class="no-margin-top">{{ title }}</h3>
         <form method="post" class="basic-form">
             {% csrf_token %}
 
-            {{ form.as_p }}
+            {% for field in form.hidden_fields %}
+                {{ field }}
+            {% endfor %}
+            {% for field in form.visible_fields %}
+               <p>
```

Comment:
Just use consistent alignment. You've got 4 spaces on line 75 and 3 spaces here. Use 4 spaces everywhere.